storybook/src/pages/amsterdam/FormPage/FormPage.tsx

```diff
@@ -56,14 +57,20 @@ export const FormPage = () => {
               </Paragraph>
               <Field>
                 <Label htmlFor="initials">Voorletters</Label>
-                <TextInput id="initials" name="initials" />
+                <TextInput id="initials" name="initials" size={8} value="M.K." />
               </Field>
               <Field>
                 <Label htmlFor="familyName">Achternaam</Label>
-                <TextInput autoComplete="family-name" id="familyName" name="familyName" />
+                <TextInput
+                  autoComplete="family-name"
+                  id="familyName"
+                  name="familyName"
+                  size={32}
+                  value="van der Linden"
+                />
               </Field>
               <FieldSet legend="Woonplaats">
-                <Radio name="city" value="amsterdam">
+                <Radio checked name="city" value="amsterdam">
```

Comment:
I don't think we should show preselected radio buttons. That prevents a user from not answering, and it makes it easier to send an incorrect answer.

Reply:
Good idea. I reverted it and added this as documentation.